addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -129,6 +129,14 @@ export class DiscussCoreCommon {
                     }
                 }
             });
+            this.busService.subscribe("mail.shortcode/insert", (datas) => {
```

Comment:
If it works, moved them outside of `messagingService.isReady` (the plan is to move most/all of them eventually)

Reply:
This code is useless, shortcode uses record/insert now!